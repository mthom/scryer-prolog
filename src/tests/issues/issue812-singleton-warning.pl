% hello
% line!

a :- b(X).