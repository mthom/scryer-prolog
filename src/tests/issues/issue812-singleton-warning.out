Warning: singleton variables X at line 3 of issue812-singleton-warning.pl
